src/meta/src/manager/catalog.rs

```diff
@@ -305,29 +305,6 @@ where
         }
     }
 
-    pub async fn create_table(&self, table: &Table) -> Result<NotificationVersion> {
```

Comment:
Why is this function unused? 🥵 Can we also remove `create_source`?

Reply:
Cuz this function is replaced by `start_create_table_procedure`/`finish_create_table_procedure`/`cancel_create_table_procedure`. Oh, so is `create_source`. I will remove this too.